package.json

```diff
@@ -7,19 +7,24 @@
     "url": "git://github.com/sandfox/aws-arn-parser.git"
   },
   "main": "src/arn.js",
+  "types": "types",
   "keywords": [
     "amazon",
     "aws",
     "parser",
     "arn"
   ],
   "directories": {
-    "lib": "lib",
+    "lib": "src",
     "test": "test"
   },
   "scripts": {
+    "dtslint": "dtslint types",
     "test": "node test/*.js"
   },
+  "devDependencies": {
+    "dtslint": "^0.4.2"
+  },
   "devDependencies": {},
```

Comment:
duplicate `devDependencies` key.

Reply:
Sheesh! Not sure how I let that one slip in. Corrected.